examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,54 @@
+  
+"""
+==============================================
+Plot Positions on a blank Map
+==============================================
+This example shows how to plot random positions on a blank map
+"""
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+
+################################################################################
+# Creating blank map data with zeros
```

Comment:
```suggestion
# First we will create a blank map using with an array of zeros.
# Since there is no WCS information, we will need to construct a header to pass to Map.
```